### ISSUE TITLE
No eyedropper icon in LOD panel

### ISSUE THREAD

**Issue by Creature40** [Sollumz/Sollumz#1004]
### 🔍 Issue Description

![Image](https://github.com/user-attachments/assets/1f1ed4b6-0c44-4717-8b97-8590f3a96433)

No eyedropper icon in LOD panel never mind if this is a clean scene or an old file

### 🛠️ Sollumz Version(s)

2.5

### 🖥️ Blender Version(s)

4.2.3

### 🔄 Reproduction steps

nothing to do here the icons just does not appear

### ⚠️ Current Result

No eyedropper icon in LOD panel

**Comment by alexguirre** [comment]
Yeah, that's expected after the changes to improve the LOD management. 

Blender add-ons UI depends a lot on the underlying code structure, and since those properties are actually strings now, not pointers to meshes, Blender doesn't show the eyedropper.

It is possible to fix, but requires contributing to Blender and them accepting it, then should start to work after some new Blender release. I did a prototype a while ago, I should pick it up again.

**Comment by alexguirre** [comment]
Picked up my changes and made some more fixes. I opened PR to Blender, lets if they accept it: https://projects.blender.org/blender/blender/pulls/129430
